tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://app.codacy.com/gh/cpuabuse/cross-echo/pullRequest?prid=5213902)